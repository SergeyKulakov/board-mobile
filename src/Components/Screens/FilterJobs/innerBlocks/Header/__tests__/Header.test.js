import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { Header as Component } from '../index'

describe('FilterJobs.Header', () => {
  const defaultProps = {
    text: {
      title: 'title',
      clearText: 'clearText',
    },
    onBackClick: () => null,
    onClearClick: () => null,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
