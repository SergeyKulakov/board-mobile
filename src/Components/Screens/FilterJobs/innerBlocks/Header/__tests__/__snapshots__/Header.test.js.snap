// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterJobs.Header render snapshot 1`] = `
<Component>
  <Styled(View)>
    <Styled(View)>
      <Styled(View)>
        <BackIcon
          onClick={[Function]}
        />
      </Styled(View)>
    </Styled(View)>
    <Styled(View)>
      <ScreenTitle
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        title
      </ScreenTitle>
    </Styled(View)>
    <Styled(View)>
      <TouchableOpacity
        activeOpacity={0.2}
        hitSlop={
          Object {
            "bottom": 15,
            "left": 15,
            "right": 15,
            "top": 15,
          }
        }
        onPress={[Function]}
      >
        <Styled(Text)>
          clearText
        </Styled(Text)>
      </TouchableOpacity>
    </Styled(View)>
  </Styled(View)>
</Component>
`;
